feat: article meta blocks EXLM-180

nitin-rachabathuni opened the pull request:
Please always provide the Jira Issue your PR is for, as well as test URLs where your change can be observed (before and after):

Jira ID: https://jira.corp.adobe.com/browse/EXLM-180

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.page/docs/integrations-learn/experience-cloud/solution-categories/content-management
- After: https://exlm-180-article-meta--exlm--adobe-experience-league.hlx.page/docs/integrations-learn/experience-cloud/solution-categories/content-management

ahmed-musallam commented:
@nitin-rachabathuni overall, I think decorating based on textContent is a bad idea.

In the converter, can we split this into 2 blocks instead? one for topic-list and one for created-for ?

nitin-rachabathuni commented:
> @nitin-rachabathuni overall, I think decorating based on textContent is a bad idea.
> 
> In the converter, can we split this into 2 blocks instead? one for topic-list and one for created-for ?

Hi @ahmed-musallam 
I had to do it based on textContent as id/classes/meta-tags are all getting stripped off. Went ahead with textContent as there is no other way. cc @thomsebastin @devisree29

nitin-rachabathuni commented:
> @nitin-rachabathuni overall, I think decorating based on textContent is a bad idea.
> 
> In the converter, can we split this into 2 blocks instead? one for topic-list and one for created-for ?

Hi @ahmed-musallam , I have updated the code, ty.

Sivaramvt commented:
@devisree29 could you please address this at the convertor side.

thomsebastin commented on blocks/article-metadata-createdby/article-metadata-createdby.css:
```diff
@@ -0,0 +1,23 @@
+
+.meta-created-for li:not(:first-child) {
+    font-size: 14px;
+    border: 1px solid #d3d3d3;
+    color: #505050;
```
Please use variables @nitin-rachabathuni

thomsebastin replied:
@nitin-rachabathuni Also please include `SCSS` file as well.
